crates/rpc/rpc/src/eth/error.rs

```diff
@@ -22,44 +22,48 @@ pub type EthResult<T> = Result<T, EthApiError>;
 
 /// Errors that can occur when interacting with the `eth_` namespace
 #[derive(Debug, thiserror::Error)]
-#[allow(missing_docs)]
 pub enum EthApiError {
     /// When a raw transaction is empty
     #[error("empty transaction data")]
     EmptyRawTransactionData,
+    /// When decoding a signed transaction fails
     #[error("failed to decode signed transaction")]
     FailedToDecodeSignedTransaction,
+    /// When the transaction signature is invalid
     #[error("invalid transaction signature")]
     InvalidTransactionSignature,
+    /// Errors related to the transaction pool
     #[error(transparent)]
     PoolError(RpcPoolError),
+    /// When an unknown block number is encountered
     #[error("unknown block number")]
     UnknownBlockNumber,
     /// Thrown when querying for `finalized` or `safe` block before the merge transition is
-    /// finalized, <https://github.com/ethereum/execution-apis/blob/6d17705a875e52c26826124c2a8a15ed542aeca2/src/schemas/block.yaml#L109>
+    /// finalized
     #[error("unknown block")]
     UnknownSafeOrFinalizedBlock,
+    /// Thrown when an unknown block or transaction index is encountered
     #[error("unknown block or tx index")]
     UnknownBlockOrTxIndex,
+    /// When an invalid block range is provided
     #[error("invalid block range")]
     InvalidBlockRange,
     /// An internal error where prevrandao is not set in the evm's environment
     #[error("prevrandao not in the EVM's environment after merge")]
     PrevrandaoNotSet,
-    /// Excess_blob_gas is not set for Cancun and above.
-    #[error("excess blob gas missing the EVM's environment after Cancun")]
+    /// `excess_blob_gas` is not set for Cancun and above
+    #[error("excess blob gas missing in the EVM's environment after Cancun")]
     ExcessBlobGasNotSet,
-    /// Thrown when a call or transaction request (`eth_call`, `eth_estimateGas`,
-    /// `eth_sendTransaction`) contains conflicting fields (legacy, EIP-1559)
```

Comment:
I want to keep the reference to the rpc calls

Reply:
done